addons/web/static/tests/fields/basic_fields_tests.js

```diff
@@ -2969,6 +2969,47 @@ QUnit.module('basic_fields', {
         form.destroy();
     });
 
+    QUnit.test('date field support internalization', function (assert) {
+        assert.expect(2);
+
+        var originalLocale = moment.locale();
+        var originalParameters = _.clone(core._t.database.parameters);
+
+        moment.defineLocale('norvegianForTest', {
+            monthsShort : 'jan._feb._mars_april_mai_juni_juli_aug._sep._okt._nov._des.'.split('_'),
+            monthsParseExact : true,
+            dayOfMonthOrdinalParse: /\d{1,2}\./,
+            ordinal : '%d.',
+        });
+
+        moment.locale('norvegianForTest');
```

Comment:
this line is not necessary i think (the call to defineLocale already sets the locale)